### PR TITLE
add admin/sales/locations UI

### LINKED ISSUE #255: Add admin/sales/locations (UI only).
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Screenshot: 
locations

### PR DESCRIPTION
**Pull request by sumedh123**
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #255 
locations

### CODE REVIEW

**Comment by sumedh123** [comment]
@geekyd Please review.

File: app/templates/admin/sales/locations.hbs [hunk 1]
```diff
@@ -1 +1,51 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header six wide column">{{t 'Tickets Summary'}}</h1>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: @sumedh123 Make this `h2` and get rid of six wide column

Comment (sumedh123) [inline app/templates/admin/sales/locations.hbs]: Done.

File: app/templates/admin/sales/locations.hbs [hunk 2]
```diff
@@ -1 +1,51 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h2 class="ui header">{{t 'Tickets Summary'}}</h2>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: @sumedh123 Please put this header inside a `column` as you would have clearly noticed, that it is no longer aligned to the table

**Comment by geekyd** [comment]
@sumedh123  Please update the branch & test for linting issue as we have added new version of ember-cli-eslint.

File: app/templates/admin/sales/locations.hbs [hunk 3]
```diff
@@ -1 +1,53 @@
...
+            <th class="right aligned">{{t 'Tickets'}}</th>
+            <th class="right aligned">{{t 'Sales'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |entry|}}
+            <tr>
+              <td>{{entry.location}}</td>
+              <td class="right aligned">{{entry.completedTickets}}</td>
+              <td class="right aligned">US${{number-format entry.completedSales}}</td>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Please Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs [hunk 4]
```diff
@@ -1 +1,53 @@
...
+              <td class="right aligned">{{entry.placedTickets}}</td>
+              <td class="right aligned">US${{number-format entry.placedSales}}</td>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Please Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs [hunk 5]
```diff
@@ -1 +1,53 @@
...
+              <td class="right aligned">{{entry.pendingTickets}}</td>
+              <td class="right aligned">US${{number-format entry.pendingSales}}</td>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs [hunk 6]
```diff
@@ -1 +1,53 @@
...
+            </tr>
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr>
+            <th><span class="weight-600">{{t 'Total'}}</span></th>
+            <th class="right aligned">{{totalCompletedTickets}}</th>
+            <th class="right aligned">US${{number-format totalCompletedSales}}</th>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Please Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs [hunk 7]
```diff
@@ -1 +1,53 @@
...
+            <th class="right aligned">{{totalPlacedTickets}}</th>
+            <th class="right aligned">US${{number-format totalPlacedSales}}</th>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs [hunk 8]
```diff
@@ -1 +1,53 @@
...
+            <th class="right aligned">{{totalPendingTickets}}</th>
+            <th class="right aligned">US${{number-format totalPendingSales}}</th>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/locations.hbs]: Leave a blank space after US$ for common style along all the tables

File: app/templates/admin/sales/locations.hbs (hunk 3 above)
Comment (sumedh123) [inline app/templates/admin/sales/locations.hbs]: Done.

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 Please review.

File: app/routes/admin/sales/locations.js [hunk 9]
```diff
@@ -5,5 +5,26 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Location');
+  },
+  model() {
+    return [{
+      location         : 'Delhi',
+      eventDate        : new Date(),
```
Comment (geekyd) [inline app/routes/admin/sales/locations.js]: Are we using `eventDate` anywhere?

Comment (sumedh123) [inline app/routes/admin/sales/locations.js]: Ohh sorry, I actually used the data of other route as it is. Corrected now.

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 @niranjan94 Please review and get it merged if no changes are needed. :)

**Comment by geekyd** [comment]
@sumedh123 Seems like some changes are still not done please make all the requested changes.

File: app/templates/admin/sales/locations.hbs (hunk 3 above)
Comment (geekyd) [inline app/templates/admin/sales/locations.hbs]: @sumedh123 These changes are still missing

**Comment by sumedh123** [comment]
@geekyd Sorry I hit ctrl+z mistakenly. That's why the spaces were removed. Please review now :).

**Comment by geekyd** [comment]
@sumedh123 Please provide deployment link. So that I can have a look at it before approving. Thanks

**Comment by sumedh123** [comment]
@geekyd https://oefrontend2017.herokuapp.com

File: app/templates/admin/sales/locations.hbs [hunk 10]
```diff
@@ -1 +1,53 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="sixteen wide column">
```
Comment (geekyd) [inline app/templates/admin/sales/locations.hbs]: Please remove this div & simply add `column` to h2.

Comment (sumedh123) [inline app/templates/admin/sales/locations.hbs]: Please check now.

File: app/templates/admin/sales/locations.hbs [hunk 11]
```diff
@@ -1 +1,51 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h2 class="ui header sixteen wide column">{{t 'Tickets Summary'}}</h2>
```
Comment (geekyd) [inline app/templates/admin/sales/locations.hbs]: @sumedh123 We do not really need to add `sixteen wide` here please remove it.

Comment (sumedh123) [inline app/templates/admin/sales/locations.hbs]: Done

**Pull request merged**